CONTRIBUTING.md

```diff
@@ -250,15 +250,9 @@ node scripts/download_dashboards.mjs
 
 ## Contributing to Documentation
 
-When submitting PRs for documentation updates, build and run the documentation locally to ensure functionality before submission. For first time documentation contributors, install the python dependencies with `yarn docs:install`. Build the documentation locally with `yarn docs:build` and serve with `yarn docs:serve`
+When submitting PRs for documentation updates, build and run the documentation locally to ensure functionality before submission. First generate the CLI documentation with `yarn docs:build`. Then in the `docs` folder, build the documentation locally with `yarn install` and serve with `yarn start`.
```

Comment:
Would it be possible to have all commands executed from root as we had previously?